pkg/controller/kieapp/defaults/defaults_test.go

```diff
@@ -6035,3 +6035,308 @@ func createKafkaJbpmObject(dateFormat string, tasksTopics string, casesTopics st
 	}
 	return &kafkaJBPMEventEmittersObject
 }
+
+func getDefaultCors() *api.CORSFiltersObject {
+	return &api.CORSFiltersObject{
```

Comment:
keep in mind that, user, besides to have the option to set custom filters he can just enable CORS with all default configuration, so, it shouldn't be needed you set all manually here.

Reply:
ok, using a flag to ask default values